spec/spec_helper_acceptance.rb

```diff
@@ -4,14 +4,15 @@
 require 'beaker/module_install_helper'
 
 run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
+install_module
+install_module_dependencies
 
-RSpec.configure do |c|
-  # Readable test descriptions
-  c.formatter = :documentation
-
-  # Configure all nodes in nodeset
-  c.before :suite do
-    install_module
-    install_module_dependencies
+# Run it twice and test for idempotency
+def apply2(pp)
```

Comment:
This looks funky.  Don't we have a 'shared example' we can use instead?

Reply:
Suggest something a bit like https://github.com/theforeman/puppet-puppet/blob/master/spec/spec_helper_acceptance.rb#L37-L45

Reply:
updated